setup.py

```diff
@@ -150,7 +147,8 @@ def _configure_for_legacy_image(self, common_args, build_dir):
     setup(
         ext_modules=[CMakeExtension("arcticdb_ext")],
         package_dir={"": "python"},
-        packages=find_packages(where="python", exclude=["tests", "tests.*"]),
+        packages=find_packages(where="python", exclude=["tests", "tests.*"])
+        + find_namespace_packages(where="python", include=["arcticdb.proto.*"]),
```

Comment:
There's a literal `+` here!

Reply:
That's the list concatenation operator? `black` forced it into that location.